### ISSUE TITLE
Windows missing /dev/app.php

### ISSUE THREAD

**Issue by hasokeric** [ericlbarnes/wardrobe#61]
I made a Windows virtual host wardrobe.dev and then when i went to the install process i hit install it did this:

File does not exist at path C:\xampp\htdocs\wardrobe\app/config/dev/app.php

then once i added the /dev/app.php i re-ran install and

array_replace_recursive(): Argument #2 is not an array in

``` php
protected function mergeEnvironment(array $items, $file)
{
   return array_replace_recursive($items, $this->files->getRequire($file));
}
```

The fix for me was to add to start/global.php

``` php
$env = $app->detectEnvironment(array(

  'local' => array('*.dev'),

));
```

**Comment by ericlbarnes** [comment]
Yes this is actually defined in [bootstrap/start.php](https://github.com/ericbarnes/wardrobe/blob/master/bootstrap/start.php#L32)

I just changed to this because I didn't think people would be using wardrobe.dev :smile:

**Comment by AliuDardan** [comment]
It's not working to me..?

File does not exist at path C:\wamp\www\cms\app/config/local/app.php  or
or
File does not exist at path C:\wamp\www\cms\app/config/production/app.php

Any idea..?

All the problem is going to be at "Wardrobe Install" so has any option to remove this steps, to configure manually the database?

**Comment by q8coder** [comment]
open https://github.com/ericbarnes/wardrobe/blob/master/bootstrap/start.php  
replace these lines 
'local' => array('localhost', '127.0.0.1'),
  'dev' => array('wardrobe.dev'),
with 
'' => array('localhost', '127.0.0.1'),
  'dev' => array('wardrobe.dev'),
it should work with you

**Comment by vnshah** [comment]
Hi Eric

Thanks for your prompt response. I tried the above fix but it still doesn't work.

Actually, I am trying to set this up on live Ubuntu (LAMP) server.

**Comment by ericlbarnes** [comment]
We are actually working on moving this to a composer package so you might be better suited to go ahead and use this one instead - https://github.com/wardrobecms/wardrobe 

Just clone that repo and run the composer install. It should move everything into place and hopefully not have the problems you are running into.

**Comment by vnshah** [comment]
Hi Eric,

This is how I fixed it (don't know if it's right way or wrong)
- Created `production` directory in `app/config`
- Copied `app/config/app.php` to `/app/config/production/app.php`
- Changed permission `chmod -R 777 production`

It works now.

Btw, I installed Wardrobe using composer!

Thanks,
Vishal

**Comment by jeremyvaught** [comment]
I did make my brand new install called wardrobe.dev and it said that app/config/local/app.php did not exist, which it doesn't. So I created the /app/config/local/ directory and copied /app/config/app.php into it. The rest of the install went just fine.

There was no mention of /dev as I see in the support here. And I also see the new github location, so just throwing this out there, but looks like a moot point.
